plugins/item_tasks/server/rest.py

```diff
@@ -36,7 +36,7 @@ def __init__(self):
         .pagingParams(defaultSort='name')
     )
     @filtermodel(model='item')
```

Comment:
It looks like the presence of this `@filtermodel` decorator before the `@autoDescribeRoute` decorator is causing the tests to fail, when `listTasks` does not have a .params`. Moving `@filtermodel` to run after `@autoDescribeRoute` fixes the issue.

However, Girder has never made a requirement that its decorators be applied in a particular order, so I think this bug should be fixed before merging. Perhaps we should also modify one of our test cases to feature a decorator between `@autoDescribeRoute` and the base function, since `@autoDescribeRoute` has so much complicated reflection behavior now.